src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -686,6 +689,8 @@ def get_created_time(self) -> ndb.DateTimeProperty:
 
   def get_age_in_seconds(self):
     current_time = datetime.datetime.utcnow()
+    if not self.get_created_time():
```

Comment:
I don't think this check is needed since `get_created_time` seems to never return None